src/util/splitSentence.ts

```diff
@@ -55,26 +55,21 @@ const splitSentence = (value: string): string[] => {
   // pattern2, multiple symbols: ?! !!! ...
   const mainSplitRegex = /[.;!?]+/g
 
-  const splitters = value.match(mainSplitRegex)
-
-  // When it cannot be split by the main spliter, spliter by ','
-  if (!splitters)
-    return value
-      .split(',')
-      .filter(s => s !== '')
-      .map(s => s.trim())
+  const sentenceSplitters = value.match(mainSplitRegex)
```

Comment:
pardon me the liberty for renaming it
but feels like stressing the fact that it's a sentence splitter(as vs `,|and` which are splitters too)